src/checkstyle/checkstyle-suppressions.xml

```diff
@@ -34,4 +34,31 @@
     <suppress checks="AbbreviationAsWordInName" files="/spring-cloud-gcp-samples/"/>
     <suppress checks="OneTopLevelClass" files="/spring-cloud-gcp-samples/"/>
 
+    <!--  Temporary suppress for spring-cloud-previews modules,
+      for lowerCamelCase abbreviation handling limitations in generated code -->
+    <suppress checks="AbbreviationAsWordInName"
```

Comment:
Since we have such a specific check, why not apply it to the entire `/spring-cloud-previews` directory to make it less verbose?

Reply:
@meltsufin Ah, @zhumin8 and I were just having this discussion above. I think either way works for unblocking the current checkstyle failures, the per-file approach ([e3d2180](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1455/commits/e3d2180330189b27bfb66446d22e3310b8e5c035)) gives better visibility of which files are affected but the blanket approach ([ca8506e](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1455/commits/ca8506e8e056ab5e5c804cf325e8f81a84ee5499)) makes the file cleaner. Do you have a strong preference here on which one to apply?

Reply:
No strong preference. This is fine.